composer.json

```diff
@@ -16,7 +16,7 @@
         "knpuniversity/oauth2-client-bundle": "^1.15",
         "league/oauth2-google": "^2.0",
         "nyholm/psr7": "^0.2.3",
-        "php-http/curl-client": "^1.7",
+        "php-http/guzzle6-adapter": "1.1.1",
```

Comment:
It's better to use `^1.1` notation to let composer update the package to minor version